src/components/UI/Notes/ArchiveNotes.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import DOMPurify from 'dompurify';
+import { BiJournalPlus, BiPatchPlus, BiUnarchive } from '../Icons/Icons';
+import useAxios from 'src/customhook/useAxios';
+import { useArchive } from 'src/context/ArchiveContext';
+import { VAR_ENCODE_TOKEN } from 'src/utils/Route';
+import { useNotes } from 'src/context/NotesContext';
+
+type Props = { props: any }
+
```

Comment:
If you're using Typescript and you know the props before hand, try using explicitly defining the props and their types. If any value has more than one type, you can use Typescript unions also.